docs/plugins/analysis-phonetic.asciidoc

```diff
@@ -17,9 +17,9 @@ The `phonetic` token filter takes the following settings:
 `encoder`::
 
     Which phonetic encoder to use.  Accepts `metaphone` (default),
-    `doublemetaphone`, `soundex`, `refinedsoundex`, `caverphone1`,
+    `double_metaphone`, `soundex`, `refined_soundex`, `caverphone1`,
     `caverphone2`, `cologne`, `nysiis`, `koelnerphonetik`, `haasephonetik`,
-    `beidermorse`, `daitch_mokotoff`.
+    `beider_morse`, `daitch_mokotoff`.
```

Comment:
If this is changes, `beidermorde` in L28 should be changed as well. Also there is an occurence in plugins/analysis-phonetic/src/test/resources/org/elasticsearch/index/analysis/phonetic-1.yml and plugins/analysis-phonetic/src/test/java/org/elasticsearch/index/analysis/AnalysisPhoneticFactoryTests.java where we should then also test with this name, although the other variant is supported as well.